http/controllers/user.controller.ts

```diff
@@ -49,14 +49,24 @@ class UserController {
 
   async deleteUserById(req: Request, res: Response) {
     const { userId } = req.params;
-    const user = await UserServiceInstance.deleteById(userId);
+    const user = await UserServiceInstance.deleteById({ id: userId });
     if (!user) {
       return res
         .status(404)
         .json({ success: false, error: JSON_MESSAGES.RESOURCE_NOT_FOUND });
     }
     return res.json({ success: true, message: JSON_MESSAGES.DELETED });
   }
+
+  async deleteAll(req: Request, res: Response) {
+    const count = await UserServiceInstance.deleteAllUsers();
+    if (!!count) {
```

Comment:
https://github.com/denysdovhan/wtfjs#true-is-false

Reply:
this is just for personal use to clean the database. would eliminate it in future